.github/workflows/tests.yml

```diff
@@ -2,9 +2,9 @@ name: Tests
 
 on:
   push:
-    branches: [ master ]
+    branches: [ main ]
```

Comment:
wouldn't we have to rename the master branch upfront?

Reply:
Yes we will!

Reply:
Done.